README.md

```diff
@@ -54,9 +54,9 @@ For more details on the yotta/mbed OS edition of mbed TLS, including example pro
 
 ### Make
 
-We intentionally only use the minimum of `Make` functionality, as a lot of `Make` features are not supported on all different implementations of Make or on different platforms. As such, the Makefiles sometimes require some manual changes or export statements in order to work for your platform.
+We require GNU Make. To build the library and the sample programs, GNU Make and a C compiler are sufficient. Some of the more advanced build targets require some Unix/Linux tools.
```

Comment:
@sbutcher-arm It is not true that “We intentionally only use the minimum of Make functionality”. We use some functionality that, as far as I know, exists only in GNU make. This functionality does not exist in Microsoft nmake, BSD pmake, Borland make, or Solaris make.

I'm not keen on the sentence about “require some manual changes or export statements”. Export statements are a GNU make feature, so you'd only need them if you were using GNU make. And anyway where would they be useful? That's not obvious to me at first glance. If there are places where we think export statements would help on some platforms, we should document it. As for needing manual changes, in principle anything can be made to work if you make enough changes... Again stating that is only useful if we guide those changes.